src/test/java/no/sikt/sws/IndexHandlerTest.java

```diff
@@ -173,25 +167,30 @@ void shuldPassQueryParams() throws IOException {
         assertThat(response.getStatusCode(), is(equalTo(HTTP_OK)));
     }
 
-    @Test
-    void shouldHandleSearchRequestWithQueryParameters() throws IOException {
+    @TestFactory
+    @DisplayName("Opensearch parameter requests")
+    @SuppressWarnings("JUnitMalformedDeclaration")
+    public Stream<DynamicTest> testRequestWithQueryParameters() {
+
+        var responseTests = getStreamOfTestCases().filter(TestCaseSws::isParamRequestTest);
```

Comment:
Is responseTests = paramRequestTest?

Reply:
`responseTests  -> requestsWithParameters`